postgres/pgdump.go

```diff
@@ -389,87 +386,88 @@ func processCopyStmt(conv *internal.Conv, n nodes.CopyStmt) *copyOrInsert {
 		// If we don't have schema information for a table, we drop all copy
 		// statements for it. The most likely reason we don't have schema information
 		// for a table is that it is an inherited table - we skip all inherited tables.
-		conv.SkipStatement(prNodes([]nodes.Node{n}))
-		internal.VerbosePrintf("Processing %v statement: table %s not found", reflect.TypeOf(n), table)
+		conv.SkipStatement(printNodeType(n))
+		internal.VerbosePrintf("Processing %v statement: table %s not found", printNodeType(n), table)
 		return &copyOrInsert{stmt: copyFrom, table: table, cols: []string{}}
 	}
 	var cols []string
-	for _, a := range n.Attlist.Items {
+	for _, a := range n.Attlist {
 		s, err := getString(a)
 		if err != nil {
-			conv.Unexpected(fmt.Sprintf("Processing %v statement Attlist: %s", reflect.TypeOf(n), err))
+			conv.Unexpected(fmt.Sprintf("Processing %v statement Attlist: %s", printNodeType(n), err))
 			s = "BOGUS_COPY_FROM_COLUMN"
 		}
 		cols = append(cols, s)
 	}
-	conv.DataStatement(prNodes([]nodes.Node{n}))
+	conv.DataStatement(printNodeType(n))
 	return &copyOrInsert{stmt: copyFrom, table: table, cols: cols}
 }
 
-func processVariableSetStmt(conv *internal.Conv, n nodes.VariableSetStmt) {
-	if n.Name != nil && *n.Name == "timezone" {
-		if len(n.Args.Items) == 1 {
-			switch c := n.Args.Items[0].(type) {
-			case nodes.A_Const:
-				tz, err := getString(c.Val)
+func processVariableSetStmt(conv *internal.Conv, n *pg_query.VariableSetStmt) {
+	if n.Name == "timezone" {
+		if len(n.Args) == 1 {
+			node := n.Args[0]
```

Comment:
Maybe arg instead of node?
node is very generic; arg gives some sense of what this represents.
Maybe even arg0 (but that's probably too long for golang var names!)

Reply:
Using `arg` makes sense since we check above for `len(n.Args) == 1`. Done @https://github.com/cloudspannerecosystem/harbourbridge/pull/172/commits/3044120ca85a3b4386f12ca5a583fe4436cd0c54.